english/wikipedia/wiki.md

```diff
@@ -12,3 +12,7 @@ You are so impressed with Wikipedia, you decide that you might even like to
 [donate](http://wikimediafoundation.org/wiki/Support_Wikipedia/en) some money to help keep Wikipedia running.
 
 Until you get bored and go explore [here](https://www.urbandictionary.com) instead.
+
+Then after you explore the website, you learn a few neat tricks. 
+
+[Play minecraft](english/you-are-in-minecraft/minecraft.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.